sdcm/tester.py

```diff
@@ -1000,7 +1000,8 @@ def get_scylla_versions(self):
         try:
             versions_output = self.db_cluster.nodes[0].remoter.run('rpm -qa | grep scylla').stdout.splitlines()
             for line in versions_output:
-                for package in ['scylla-jmx', 'scylla-server', 'scylla-tools']:
```

Comment:
How about replacing the fixed package name with pattern?

```
+++ b/sdcm/tester.py
@@ -1003,7 +1003,7 @@ class ClusterTester(Test):
         try:
             versions_output = self.db_cluster.nodes[0].remoter.run('rpm -qa | grep scylla').stdout.splitlines()
             for line in versions_output:
-                for package in ['scylla-jmx', 'scylla-server', 'scylla-tools']:
+                for package in ['scylla[-\w]*-jmx', 'scylla[-\w]*-server', 'scylla[-\w]*-tools']:
                     match = re.search('(%s-(\S+)-(0.)?([0-9]{8,8}).(\w+).)' % package, line)
                     if match:
                         versions[package] = {'version': match.group(2),
```

Or get the right package prefix by scylla_pkg()?
```
+++ b/sdcm/tester.py
@@ -1003,7 +1003,8 @@ class ClusterTester(Test):
         try:
             versions_output = self.db_cluster.nodes[0].remoter.run('rpm -qa | grep scylla').stdout.splitlines()
             for line in versions_output:
-                for package in ['scylla-jmx', 'scylla-server', 'scylla-tools']:
+                for package in ['-jmx', '-server', '-tools']:
+                    package = self.db_cluster.nodes[0].scylla_pkg() + package
                     match = re.search('(%s-(\S+)-(0.)?([0-9]{8,8}).(\w+).)' % package, line)
                     if match:
                         versions[package] = {'version': match.group(2),
```